docker-compose-1.yml

```
@@ -0,0 +1,67 @@
+              NODE_ENV:
+              MONGODB_URI: mongodb://mongo:27017/nodegoat
+            command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+            ports:
+              - "4000:4000"
+
+          mongo:
+            image: mongo:4.4
+            user: mongodb
+            expose:
+              - 27017
+            image: mongo:4.4
```

YAMLlint (1.37.1)

[error] duplication of key "image" in mapping

(key-duplicates)